docs/implicit-creation-options.md

```diff
@@ -61,21 +61,21 @@ example, if you wanted all Fakes to be Strict, you might write
 something like this:
 
 ```csharp
-class MakeEverythingStrictOptionsBuilder: IFakeOptionsBuilder
+class MakeEverythingStrictOptionsBuilder : IFakeOptionsBuilder
 {
     public bool CanBuildOptionsForFakeOfType(Type type)
     {
         return true;
     }
 
-    public void BuildOptions(Type typeOfFake, IFakeOptions options);
+    public void BuildOptions(Type typeOfFake, IFakeOptions options)
     {
-        options => options.Strict();
+        options.Strict();
     }
 
-    public int Priority
+    public Priority Priority
     {
-        get { Priority.Default; } // equivalent to value 0
+        get { return Priority.Default; }

```

Comment:
I think it's a good idea to retain the "equivalent to 0", or to note that up somewhere in else in the article, as was done in [Formatting Argument Values](http://fakeiteasy.readthedocs.org/en/latest/formatting-argument-values/) (but not, I see in [Custom Dummy Creation](http://fakeiteasy.readthedocs.org/en/latest/custom-dummy-creation/)) so users know what they're overriding, and that supplying a value of `0` won't help them if they are competing with another custom implementation that's using `Priority.Default`.
If you agree, we should probably fix Custom Dummy Creation. If not, we should fix Formatting Argument Values.